docs/gates.md

```diff
@@ -1,58 +1,235 @@
-version of a gate is not used when sending the circuit to a
-quantum computer for execution, but it can be used with
-various simulators. See [noise documentation](noise.md).
+## Other gate features
 
+Most ``Gate``s operate on a specific number of qubits, which can be accessed
+by the ``num_qubits()`` function.  One notable exception is the
+``MeasurementGate`` which can be applied to a variable number of qubits.
 
-### Magic Methods
+Most gates also have a unitary matrix representation, which can be accessed
+by ``cirq.unitary(gate)``.  
+
+Not all `Gate`s correspond to unitary evolution. They may represent a
+probabilistic mixture of unitaries, or a general quantum channel. The component
+unitaries and associated probabilities of a mixture can be accessed by
+`cirq.mixture(gate)`. The Kraus operator representation of a channel can be
+accessed by `cirq.channel(gate)`. Non-unitary gates are often used in the
+simulation of noise. See [noise documentation](noise.md) for more details.
+
+Many arithmetic operators will work in the expected way when applied to
+gates.  For instance, ``cirq.X**0.5`` represents a square root of X gate.
+These can also be applied to Operators for a more compact representation, such
+as ``cirq.X(q1)**0.5`` will be a square root of X gate applied to the q1
+qubit.  This functionality depends on the "magic methods" of the gate
+being defined (see below for details).
+
+Gates can be converted to a controlled version by using `Gate.controlled()`.
+For instance, `cirq.X.controlled()` returns a `cirq.CNOT` gate.
+Operations have similar functionality `Operation.controlled_by()`, such as
+`cirq.X(q0).controlled_by(q1)`.
+
+## Common gates
+
+Cirq supports a number of gates natively, with the opportunity to extend these
+gates for more advanced use cases.
+
+### Measurement gate
+
+**cirq.MeasurementGate** This is a measurement in the computational basis.
+This gate can be applied to a variable number of qubits.  The function
+`cirq.measure(q0, q1, ...)` can also be used as a short-hand to create a
+`MeasurementGate` .
+
+### Single qubit gates
+
+Most single-qubit gates can be thought of as rotation around an axis in the
+[Bloch Sphere](https://en.wikipedia.org/wiki/Bloch_sphere) representation and
+are usually referred to by their axis of rotation.
+
+
+**cirq.Z / cirq.ZPowGate / cirq.rz** Rotations about the Pauli ``Z`` axis.
+The matrix of `cirq.Z**t` and the equivalent representation
+`cirq.ZPowGate(exponent=t)` is ``exp(i pi |1⟩⟨1| t)`` whereas the matrix of
+`cirq.rz(θ)` is `exp(-i Z θ/2)`.  Since computation is often done in the
+Z-basis, these are implemented as phase changes on later operations on
+[Google devices](google/devices.md) instead of a physical modification applied
+to the qubits.
+
+**cirq.Y / cirq.YPowGate / cirq.ry** Rotations about the Pauli ``Y`` axis.
+The gate `cirq.Y**t` and the equivalent representation
+`cirq.YPowGate(exponent=t)` are measured in terms of 180 degree turns, while
```

Comment:
```suggestion
`cirq.YPowGate(exponent=t)` are measured in terms of 180 degree turns (half turns), while
```
since a lot of documentation uses the half-turn terminology.

Reply:
Also changed on Z gate explanation.